dev/io.openliberty.restfulWS.3.0.cdi.3.0_fat/test-applications/basic/src/io/openliberty/restfulWS30/cdi30/fat/basic/HelloWorldResource.java

```diff
@@ -0,0 +1,171 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.restfulWS30.cdi30.fat.basic;
+
+import java.util.Iterator;
+import java.util.Set;
+
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+
+import jakarta.enterprise.inject.Any;
+import jakarta.enterprise.inject.spi.Bean;
+import jakarta.enterprise.inject.spi.BeanManager;
+import jakarta.enterprise.util.AnnotationLiteral;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.core.Context;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.UriInfo;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.QueryParam;
+
+public class HelloWorldResource {
+
+    /**
+     * A static variable to hold a message. Note that for this sample, the field
+     * is static because a new <code>HelloWorldResource</code> object is created
```

Comment:
```suggestion
     * is static because a new {@code HelloWorldResource} object is created
```

I doubt if this matters - javadoc for a test, but code snippets usually need to be inside `{@code ... }` blocks instead of XML `<code>...</code>` blocks.